app/controllers/providers/application_merits_task/statement_of_cases_controller.rb

```diff
@@ -50,13 +50,13 @@ def error_message
       end
 
       def files_deleted_message(deleted_file_name)
-        I18n.t('activemodel.attributes.application_merits_task/statement_of_case.file_deleted', file_name: deleted_file_name)
+        I18n.t('activemodel.attributes.uploaded_evidence_collection.file_deleted', file_name: deleted_file_name)
```

Comment:
It totally makes sense to have a single version of this text, but if you are no longer using the `application_merits_task/statement_of_case` or `gateway_evidence` file_deleted entries, can they be deleted from the locale file?